src/Analyzers/CSharp/Tests/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -7622,6 +7629,29 @@ public List<UU> M<TT, UU>(Dictionary<TT, List<UU>> a, TT b, UU c) where UU : TT
     [Fact, WorkItem("http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/994328")]
     public async Task TestDisposePatternWhenAdditionalUsingsAreIntroduced2()
     {
+        const string equatableParameter =
+#if NET9_0_OR_GREATER
+            "[AllowNull] int other"
```

Comment:
this seems... undesirable.  why have `AllowNull` on something known to be a value type?

Reply:
I'll crosscheck with the explicit interface implementation refactoring to see whether this is the existing behavior

Reply:
waiting on this.

Reply:
Check the test `GenericInterfaceNotNull1` in `ImplementInterfaceTests`. It generates `[AllowNull] int bar`, with the type of the parameter in the interface method being a generic type parameter, which is a similar case to this. I don't think it's that big of an issue.